functional.py

```diff
@@ -0,0 +1,124 @@
+def sequential_map(*args):
+    """
+    The function implements sequentially any number of functions on the container
+    :param args:
+    :return con:
+    """
+    funcs = args[:-1]
+    container = args[-1]
+    for i in range(len(funcs)):
+        container = list(map(funcs[i], container))
+    res = container
+    print(res)
+
+
+def consensus_filter(*args):
+    """
+    The function returns a list of values that give True when the container is passed.
+    :param args:
+    :return :
+    """
+
+    result = []
+    ind_res = []
+    index_list = []
+    passed_list = []
+
+    funcs = args[:-1]
+    container = args[-1]
+    for i in range(len(funcs)):
+        res = tuple(map(funcs[i], container))
+        result.append(res)
+
+    for line in result:
+        for i in range(len(line)):
+            if line[i] is True:
+                index_list.append(i)
+
+    for index in index_list:
+        if index_list.count(index) == len(funcs) and index not in ind_res:
+            ind_res.append(index)
+
+    for index in ind_res:
+        res = container[index]
+        passed_list.append(res)
+    print(passed_list)
```

Comment:
Очень много лишнего кода, всё сильно проще, надо только вспомнить про функцию `filter`
```suggestion
    *funcs, container = args
    for func in funcs:
        container = filter(func, container)
    return list(container)
```